controllers/managedcluster.go

```diff
@@ -53,14 +66,34 @@ func getKlusterletAddonConfig() *unstructured.Unstructured {
 					"enabled": false,
 				},
 				"certPolicyController": map[string]interface{}{
-					"enabled": true,
+					"enabled": grcEnabled,
 				},
 			},
 		},
 	}
 	return klusterletaddonconfig
 }
 
+func IsKlusterletAddonConfigUpdated(newKlusterletaddonconfig, klusterletaddonconfig *unstructured.Unstructured,
```

Comment:
Could we make this a provide function?
```suggestion
func isKlusterletAddonConfigUpdated(newKlusterletaddonconfig, klusterletaddonconfig *unstructured.Unstructured,
```